syft/src/main/java/org/openmined/syft/domain/JobRepository.kt

```diff
@@ -136,7 +141,7 @@ internal class JobRepository(
                 .flatMap { modelInputStream ->
                     jobLocalDataSource.saveAsync(
                         modelInputStream,
-                        "${config.filesDir}/models",
+                        jobLocalDataSource.getModelsPath(config),
```

Comment:
This info is already in the data source. The repository should not need to pass it again.

Reply:
Yeah better would be to pass the config object and store it alongside for future use